### PR TITLE
sdllopan, fix reference to libSDL, cleanup

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: games-puzzle/sdllopan/sdllopan-10.recipe [hunk 1]
```diff
@@ -26,21 +25,20 @@ ARCHITECTURES="x86_gcc2 x86_64"
 
 PROVIDES="
 	sdllopan = $portVersion
-	app:lopan = $portVersion
+	app:Lopan = $portVersion
```
Comment (Begasus) [inline games-puzzle/sdllopan/sdllopan-10.recipe]: Is there any way to set this to "SDL Lopan"?

Comment (pulkomandy) [reply games-puzzle/sdllopan/sdllopan-10.recipe]: Why? The executable and deskbar entry are called just Lopan and that's how it should be. The end-user doesn't need to know sdl is involved here.

Comment (Begasus) [reply games-puzzle/sdllopan/sdllopan-10.recipe]: I guess a picture tells more then words (check window title) :)
SDL-Lopan

**Comment by Begasus** [comment]
As a sidenote ... this also doesn't launch from Deskbar ...

File: games-puzzle/sdllopan/sdllopan-10.recipe (hunk 1 above)
Comment (korli) [reply games-puzzle/sdllopan/sdllopan-10.recipe]: app:SDL\ Lopan doesn't work as expected?

Comment (Begasus) [reply games-puzzle/sdllopan/sdllopan-10.recipe]: Need to check :)

Comment (Begasus) [reply games-puzzle/sdllopan/sdllopan-10.recipe]: Looks like it doesn't, maybe I'll just use SDL_Lopan:
```
Warning: failure creating the build package: Parse error in .PackageInfo(23:12) -> invalid character in resolvable name
```

Comment (Begasus) [reply games-puzzle/sdllopan/sdllopan-10.recipe]: I guess we should poke @scottmc for creating the window tittle? ;)

**Comment by Begasus** [comment]
@scottmc since you changed the window tittle on this one, should I go for the window tittle for the app or just leave it as "Lopan"?

**Comment by scottmc** [comment]
> 
> 
> @scottmc since you changed the window tittle on this one, should I go for the window tittle for the app or just leave it as "Lopan"?

Fix it to be just Lopan as Pulkomandy suggests above.

**Comment by Begasus** [comment]
Ok, will do :)

**Pull request merged**